### PR TITLE
Fix GHA timeout on firefox test

### PR DESCRIPTION
**Pull request by ryanking13**
For some reason, the recent playwright-python update (`v.1.23.0`) made this test slower on firefox, which results in a CI timeout.

### CODE REVIEW

File: src/tests/test_typeconversions.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 
 @given(s=text())
-@settings(deadline=4000)
+@settings(deadline=15000)
```
Comment (hoodmane) [inline src/tests/test_typeconversions.py:17]: Is this change still needed?

Comment (ryanking13) [reply src/tests/test_typeconversions.py:17]: Nop, it is not needed.

**Review by hoodmane — APPROVED** [review]
